### ISSUE TITLE
[Bug] [A11Y][Text Input] Error and Loading status are not read by Voice Over

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-ios#1057]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 14+

### OS Version

iOS 26

### App version

0.20.0

### App build number

1760480434

### App build type

beta

### Describe the issue

Error and Loading status are not vocalized by Voice Over

### Expected Behavior

Error and Loading status should be heard when text input is focused by Voice Over

### Accessibility settings

Voice Over

**Comment by pylapp** [comment]
@pya35 I will check that soon
